package.json

```diff
@@ -493,12 +493,14 @@
         "js-yaml": "^3.8.2",
         "k8s": "^0.4.12",
         "lodash": ">3",
+        "node-yaml-parser": "^0.0.6",
         "opn": "^5.2.0",
         "pluralize": "^4.0.0",
         "shelljs": "^0.7.7",
         "tmp": "^0.0.31",
         "uuid": "^3.1.0",
         "vscode-extension-telemetry": "^0.0.6",
+        "vscode-uri": "^1.0.1",
```

Comment:
Should we be declaring a package dependency on the RedHat YAML extension?

Reply:
Sure